routes/index.js

```diff
@@ -9,38 +11,43 @@ function appendLocals(req, res) {
 
     res.locals.fullUrl = proto + '://' + req.hostname + req.path;
 
+    res.locals.theme = req.query.theme;
+
+    res.locals.displayTitle = function(subtitle) {
+        return TITLE + ': ' + subtitle + ' by MaxCDN';
```

Comment:
BootstrapCDN by MaxCDN - Quick Start

Soon it will be `BootstrapCDN by StackPath` but until further notice.